core/templates/dev/head/profile/Preferences.js

```diff
@@ -32,6 +32,13 @@ oppia.controller('Preferences', [
       });
     };
 
+    var _forceDOMRefresh = function() {
+      $scope.isClean = false;
```

Comment:
I think that this variable name is a bit too vague -- what is it that is clean? Perhaps we can rename this to

    $scope.select2DropdownIsShown

and the function to

    _forceSelect2Refresh?